docs/source-app/workflows/byoc/index.rst

```diff
@@ -82,40 +307,103 @@ Parameters
+
+.. warning::
 
-Deletes a Lightning BYOC cluster. Lightning AI removes cluster artifacts and any resources running on the cluster.
+    Using the ``--force`` parameter when deleting a cluster does not clean up any resources managed by Lightning AI. Check your cloud provider to verify that existing cloud resources are deleted.
+
+    Deletion permanently removes not only the record of all runs on a cluster, but all associated artifacts, metrics, logs, etc.
+
+    This process may take a few minutes to complete, but once started it CANNOT be rolled back. Deletion permanently removes not only the BYOC cluster from being managed by Lightning AI, but tears down every BYOC resource Lightning AI managed (for that cluster id) in the host cloud. All object stores, container registries, logs, compute nodes, volumes, etc. are deleted and cannot be recovered.
+
+Use the following command to delete a Lightning cluster:
+
+.. code:: bash
+
+    lightning delete cluster <cluster-name>
+
+Here’s an example:
+
+.. code:: bash
 
-.. warning:: Using the ``--force`` parameter when deleting a cluster does not clean up any resources managed by Lightning AI. Check your cloud provider to verify that existing cloud resources are deleted.
+    lightning delete cluster my-byoc-cluster-01
+
+----
+
+*************************************************
+Step 4: Run an App on your Lightning BYOC Cluster
+*************************************************
+
+After starting an App on your cluster, you can also view the App's status.
+
+----
+
+Who should run an App?
+^^^^^^^^^^^^^^^^^^^^^^
+
+Your cluster is now up and running, so researchers, data scientists, Lightning App creators, and more, can now start running and monitoring Apps on the clusters.
+
+----
+
+Use the Lightning BYOC Cluster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once your cluster is running, you can run any Lightning App on your cluster. To run an App on the Lightning BYOC cluster, use --cloud –cluster <cluster name> in the command:
+
+.. code:: bash
+
+    lightning run app app.py —-cluster <cluster-name> —-cloud
+
+Here’s an example:
+
+.. code:: bash
 
-Deletion permanently removes not only the record of all runs on a cluster, but all associated artifacts, metrics, logs, etc.
+    lightning run app app.py —-cluster my-byoc-cluster-01 –cloud
 
-.. warning:: This process may take a few minutes to complete, but once started it CANNOT be rolled back. Deletion permanently removes not only the BYOC cluster from being managed by Lightning AI, but tears down every BYOC resource Lightning AI managed (for that cluster id) in the host cloud. All object stores, container registries, logs, compute nodes, volumes, etc. are deleted and cannot be recovered.
+Once you attempt to run the App on the cluster, you can also view the status of your App using the following command:
 
 .. code:: bash
 
-   lightning delete cluster <cluster-name>
+    lightning list apps
```

Comment:
Given the example above, this should include the cluster id:

`lightning list apps --cluster-id <cluster-id>`

without it, it would list the apps in the lightning cluster afaik.

Reply:
@nmiculinic please verify